django/test/testcases.py

```diff
@@ -825,6 +825,10 @@ class TransactionTestCase(SimpleTestCase):
     # are not available, we allow queries to be run.
     allow_database_queries = True
 
+    @classmethod
```

Comment:
Sure. Sometimes I mark a ticket RFC if only relatively trivial updates remain. Then I'll just make those updates myself if the contributor doesn't do so.